SpatialGDK/Documentation/content/actor-readiness.md

```diff
@@ -0,0 +1,70 @@
+<%(TOC)%>
+
+# Actor readiness
+Before executing any logic that requires or affects a replicated Actor, you must ensure that the Actor is _ready_. What constitutes an Actor’s “readiness” depends on what you want to do with the Actor. When using the GDK, you must define your own validity checks to signal that an Actor is ready.
+
+## Background
+When you use native Unreal networking, you cannot guarantee that a client receives Actors in a specific order (for example, the PlayerController before the Pawn). However, in ideal network conditions, the order in which a client receives Actors is usually the same. 
+
+This means you might develop your game with assumptions about when Actors are ready to interact with on clients without creating validity checks to ensure this. (The order in which clients receive Actors can change in adverse network conditions, such as high latency or packet loss, but during development or local playtests these networking issues often don’t arise.)
+
+In the GDK, [server-worker and client-worker instances]({{urlRoot}}/content/glossary#workers) communicate with each other via SpatialOS. Unlike native Unreal, where Actors arrive on clients in a similar order each time in stable networking conditions, SpatialOS can cause Actors to arrive in a different order, with different latencies each time you run your game. This means you need to define validity checks to determine when an Actor is ready to interact with.
+
+## Defining validity checks
+A validity check could be, for example, whether the Actor exists on the client-worker instance, whether a particular reference within the Actor is valid (for example the Pawn having a reference to the PlayerController), or whether a setup function has run.
+
+If you are porting your game from native Unreal to the GDK and you have not created validity checks to ensure that Actors are ready, you might find that certain behaviors don't work with SpatialOS, even though they always worked in native Unreal.
+
+### Examples 
+The following examples show you how to define validity checks to signal that an Actor is ready, for various use cases.
+
+#### When can I modify my Actor on the server-worker instance?
+
+Whenever you want to modify an Actor, you _must_ check that the server-worker instance has [authority]({{urlRoot}}/content/glossary#authority) over the Actor. The server-worker instance has authority over an Actor if the Actor's `Role` is set to `ROLE_Authority`.
+
+In the GDK, when an Actor is created on the server-worker instance:
+ 
+0. The server-worker instance has authority over the Actor for the duration of the Actor's `BeginPlay` call.
```

Comment:
Are we intending this list to start at 0.? Shouldn't it be 1?

Reply:
I _believe_ it works in the docs, whether you put 0 or 1 for all the numbered steps (it automatically increments the numbers, starting at 1). Good spot though!